### DISCUSSION TITLE
COVID-19 Integrated Surveillance Data in Italy (ISS)

### DISCUSSION THREAD

**Discussion by InterdisciplinaryPhysicsTeam** [owid/covid-19-data#2168] [Ideas]
Here is the [GitHub repository](https://github.com/InPhyT/COVID19-Italy-Integrated-Surveillance-Data) containing the Italian COVID-19 integrated surveillance incidence data ([ISS](https://www.iss.it)) disaggregated by sex, age and region.

Here is the [presentation tweet](https://twitter.com/In_Phy_T/status/1466366138246443008?s=20). 

Hope it's helpful. 

Feel free to reply to this issue or open an issue on our repository in case some requirements were unmet.

Thank you very much for your amazing work!

### Contacts 

| Author | GitHub | Twitter | 
|--------|--------|--------|
| Claudio Moroni | [@ClaudMor](https://github.com/ClaudMor) | [@Claudio__Moroni](https://twitter.com/Claudio__Moroni) | 
| Pietro Monticone | [@pitmonticone](https://github.com/pitmonticone) | [@PietroMonticone](https://twitter.com/PietroMonticone) |

**Comment by edomt** [comment]
Hi @InterdisciplinaryPhysicsTeam 
Thank you for letting us know! Very useful project 🙂
We don't currently present disaggregated case data in our files, but in case we do in the future, I'll leave this open as a discussion

**Reply by InterdisciplinaryPhysicsTeam** [reply]
Hi @edomt, 

As far as we can see OWID uses [data from Civil Protection](https://github.com/CSSEGISandData/COVID-19#:~:text=COVID%2D19/general-,Italy,-%3A), which are affected by a (variable) notification delay. The incidence data reported above are organized by date of event so they don't suffer from that delay. Would an aggregated version of our dataset already fit your needs?

**Reply by edomt** [reply]
Yes, but that's because we source all our data on cases and deaths [from Johns Hopkins University](https://github.com/CSSEGISandData/COVID-19). And we report this data without alterations. You could try to pitch the idea of a change of source to them here: https://github.com/CSSEGISandData/COVID-19/issues
